etl/steps/data/garden/who/2024-07-26/mortality_database.meta.yml

```diff
@@ -4,7 +4,10 @@ definitions:
     presentation:
       topic_tags:
         - Causes of Death
-
+    processing_level: minor
+    description_key: ["The ICD 10 codes that define this cause of death are << icd10_codes  >>."]
+  footnote: |-
+    <% if age == "Age-standardized" %>To allow for comparisons between countries and over time, this metric is [age-standardized](#dod:age_standardized).<%- endif -%>
```

Comment:
[Not a comment] This is a general question I personally have but - this does not show up in the grapher automatically, right? I checked this for the age-standardized indicators in the staging server and the footer is not prepopulated. Or is this a "WIP"-feature?

Reply:
Ah no, you are totally right, I applied this wrong 🤦🏼‍♀️, good catch!

Reply:
Oh really? Because this happened to me before as well and since the footer box in grapher doesn't have the little clickable link I always assumed this just doesn't work (yet 😅) Could you tell me how you fixed it?

Reply:
It should be there now 🙏🏼

Reply:
Ah in this case I was using the wrong index name to make the Jinja work. I think it depends a little on whether there is something in the footnote already, if there is it won't be overwritten. 

However, it does seem a little bit unpredictable when it works and when it doesn't. I'll take a look into it. Do you have an example to hand of when it didn't work for you?

Reply:
Ah, that makes sense! I think in all the cases I tried it, there was a footnote already on the chart and I could not get the one from the etl to show up 😅 I will keep that in mind though - if I run into another issue with it I'll let you know!